cmd/agent/subcommands/flare/command_test.go

```diff
@@ -86,6 +111,8 @@ func TestCommandTestSuite(t *testing.T) {
 
 func (c *commandTestSuite) TestReadProfileData() {
 	t := c.T()
+	c.startTestServers()
```

Comment:
Couldn't we add this call in the test suite `SetupSuite` function?

Reply:
It was previously in SetupSuite, but in the previous PR, I was advised to do this by [brycekahle].